easybuild/easyconfigs/a/adios/adios-2.6.0-CrayGnu-20.08.eb

```diff
@@ -1,4 +1,4 @@
-# contributed by Jean Favre (CSCS)
+# contributed by Jean Favre (CSCS); extended by Samuel Omlin (CSCS)
```

Comment:
I've just noticed that the file name does not match the toolchain (case): CrayGNU

Reply:
@omlins Can you fix the filename? Thanks

Reply:
Done.